weaviate/connection/rest.go

```diff
@@ -53,6 +54,39 @@ func NewConnection(scheme string, host string, httpClient *http.Client, headers
 	return connection
 }
 
+// WaitForWeaviate waits until weaviate is started up and ready
+func (con *Connection) WaitForWeaviate(startupTimeout time.Duration) error {
+	if startupTimeout < 0 {
```

Comment:
I think 0 would be "don't wait at all"?

Reply:
Ok, if that is desired behavior, let's make that explicit. Also, the below error message should be changed, because it indicates that `0` is not a valid value